### ISSUE TITLE
Initial screen looks unhappy :-(

### ISSUE THREAD

**Issue by probonopd** [lampix-org/lampixjs-core#1]
This is the first thing we see when launching the app for the first time.

![](https://raw.githubusercontent.com/AppImage/appimage.github.io/8b07f9d6baac25166e006d23289a7eb427ad3b39/database/LampixJS/screenshot.png)

Looks like a sad error?!

**Comment by probonopd** [comment]
Doesn't look all too great in a software catalog:
https://appimage.github.io/LampixJS/

**Comment by DSBalaban** [comment]
Hi, @probonopd

Thanks for reaching out!

This issue would be more suited on [the simulator repo](https://github.com/lampix-org/simulator). But since we're here...

Thanks for the link to the appimage software catalog, I'll see what I can do about that image.
Regarding the "error", it's simply the simulator being "sad" due to the lack of open simulations, which you can create by following the subheading instructions (create a simulation either manually, via URL, or by dragging and dropping an HTML file in the simulator).

Once open, the simulator should look as seen below.

Without simulations open:
![screenshot from 2019-01-11 15-52-26](https://user-images.githubusercontent.com/9392041/51037471-0051ba80-15b9-11e9-89f7-e34770634a2b.png)

With one simulation open:
![screenshot from 2019-01-11 15-52-40](https://user-images.githubusercontent.com/9392041/51037472-00ea5100-15b9-11e9-98ea-8b4760747a57.png)

If you keep seeing the image you linked above, could you please provide information about your system, such as the operating system name, the arch (ia32 or 64) and the resolution of your monitor? I'd like to get to the bottom of that.

**Comment by DSBalaban** [comment]
@probonopd I've created a pull request to fix the screenshot at AppImage/appimage.github.io#1138. Checked the contributing guidelines, but didn't see anything on editing. Let me know if I need to change anything.

**Comment by probonopd** [comment]
By default when one launches the app then no simulations are apparently open.
In this case (which is what every user sees when they launch the app for the first time) the app should not look so "unhappy" imho.

It is not possible to just change a screenshot, as the screenshot it automatically generated then the automated test is run. If the application has been changed to no longer be unhappy, then simply edit https://github.com/AppImage/appimage.github.io/blob/master/data/LampixJS a little bit (e.g., add a new line with a `#` in it) and send a PR on that file to trigger a new run of the test.

**Comment by DSBalaban** [comment]
I assume the screenshot is also taken automatically. It doesn't exactly look like it respects the bounds of the simulator window, since the provided screenshot looks like a 559px x 449px square picture with the center set in the center of the actual app. 

While I'm fine with removing the sadness, the image size issue remains (hence my hopeful pull request), and from what I've seen, the simulator is the only app with such a screenshot. The only question is **why**?

**Comment by probonopd** [comment]
> I assume the screenshot is also taken automatically

Yes, if no  other screenshot is provided in an AppStream metainfo file inside the AppImage.

> The only question is why?

The other screenshots that have not been taken by the automated test have been specified in an AppStream metainfo file inside the respective AppImages.

References:
- https://docs.appimage.org/packaging-guide/appstream.html
- http://output.jsbin.com/qoqukof

Please let me know once you have released an AppImage that contains AppStream metainfo, then I can pull it again.

**Comment by probonopd** [comment]
That being said, wouldn't it be cool if it would load an example simulation by default? This is the first thing the user sees when opening your app for the first time. First impressions count :+1:

**Comment by DSBalaban** [comment]
Thanks for the links! I'll get to it as soon as I can.

> Please let me know once you have released an AppImage that contains AppStream metainfo, then I can pull it again.

Will do. 

> That being said, wouldn't it be cool if it would load an example simulation by default? This is the first thing the user sees when opening your app for the first time. First impressions count

I agree. Would love to get around to improving the UX to that end.
